fix: use heck instead of covert_case for Train-Case

niladrih opened the pull request:
The Train-Case implementation on the covert_case crate seems to convert 'ma8' to 'Ma-8'. This isn't the expected result. Heck seems to get it right, 'Ma8'.

tiagolobocastro commented on call-home/Cargo.toml:
```diff
@@ -56,3 +55,4 @@ mime = "0.3.17"
 
 # parse prometheus output
 prometheus-parse = "0.2.4"
+heck = "0.5.0"
```
Please use the version 0.4.1

niladrih replied:
Done.

tiagolobocastro commented on constants/Cargo.toml:
```diff
@@ -7,4 +7,4 @@ edition = "2021"
 utils = {path = "../dependencies/control-plane/utils/utils-lib" }
-convert_case = "0.4.0"
\ No newline at end of file
+heck = "0.5.0"
```
same here

niladrih replied:
Done.

niladrih committed:
fix: use heck instead of covert_case for Train-Case
The Train-Case implementation on the covert_case crate seems to convert 'ma8' to 'Ma-8'.
This isn't the expected result. Heck seems to get it right, 'Ma8'.

Signed-off-by: Niladri Halder <niladri.halder26@gmail.com>

niladrih commented:
bors merge

niladrih commented:
bors merge

niladrih commented:
bors merge

niladrih commented:
bors merge
